src/Yesod/Markdown.hs

```diff
@@ -79,13 +80,13 @@ markdownField = Field
     , fieldEnctype = UrlEncoded
     }
 
-markdownToHtml :: Markdown -> Html
-markdownToHtml = writePandoc yesodDefaultWriterOptions
+markdownToHtml :: Markdown -> Either PandocError Html
+markdownToHtml = (writePandoc yesodDefaultWriterOptions <$>)
```

Comment:
I believe this is causing some build failures. Since you're partially applying anyway, I'd probably use `fmap`. Otherwise, you'll need another CPP-guarded import.